src/IntaRNA/Accessibility.cpp

```diff
@@ -114,4 +114,57 @@ writeRNAplfold_text( std::ostream& out, const E_type RT, const bool writeProbs )
 
 ////////////////////////////////////////////////////////////////////
 
+void
+Accessibility::
+compute_accessible_ranges( IndexRangeList& ranges, const size_t max_seq_length, E_type threshold, int i, int j) const
```

Comment:
Hi @agawrons, just to ensure I get the idea right:

this function will generate all ranges that have an ED below `threshold`, which also holds for all covered subranges.

right?